src/patternfly/demos/DataList/data-list-page-nav.hbs

```diff
@@ -1,4 +1,4 @@
-{{#> nav nav--attribute=(concat 'id="' page--id '-primary-nav" aria-label="Grouped Nav Example"')}}
+{{#> nav nav--attribute=(concat 'id="' page--id '-primary-nav" aria-label="Global"')}}
```

Comment:
`aria-label` appears to already be defined on the `nav` handlebar. It's showing up twice in the Page and Datalist demos.

Reply:
Nice find! Looks like we should just remove any reference to `aria-label` from all of the nav components in the examples, except for the tertiary nav used in the page expandable demo, data list demo, and data table demos. And in those cases, define `aria-label` as an attribute of the hbs partial, not in the `nav--attribute` attribute value, since this is the logic that generates `aria-label` in `nav.hbs`

```
<nav class="pf-c-nav{{#if nav--modifier}} {{nav--modifier}}{{/if}}"
  {{#if aria-label}}
    aria-label="{{aria-label}}"
  {{else}}
    aria-label="Global"
  {{/if}}
  {{#if nav--attribute}}
    {{{nav--attribute}}}
  {{/if}}>
  {{> @partial-block}}
</nav>
```

Reply:
Or alternatively, we could remove the `aria-label` logic from `nav.hbs`, and include it in `nav--attribute` instead. That would probably be more consistent with our other partials.